### ISSUE TITLE
How to use with others roms?

### ISSUE THREAD

**Issue by salvadorc17** [rephus/achievement-emulator#3]
Hi this project looks interesting to be javascript based, now my question is how to load roms, should include all those in website server files??

**Comment by rephus** [comment]
This emulator indeed support multiple ROMs from NES, but you might want to look at the original source of JSNES here: https://github.com/bfirsh/jsnes .

I decided not to include any more roms on this project because mapping all the trophies and memory details requires a lot of work.

**Comment by salvadorc17** [comment]
Yes i  understand that, but want to know if will needed to host all roms and this source files in server to be able to host retrophies site.

**Comment by rephus** [comment]
You can load roms in any way you want.  You can upload the roms along with the source, or have them on a separate URL (eg: S3) or even request the user to upload their own (might need to change the code in order to do that). This is the code where the game is loaded from the URL: https://github.com/bfirsh/jsnes/blob/master/source/ui.js#L180

In the main README, you can see how to initialize this project: 
    
    Create a folder called roms and add the mario.nes rom (USA)

They were not included for copyright reasons, but I'm sure you can find it on Internet.

**Comment by salvadorc17** [comment]
I do have codes for some other games, you would be interested in add support for those?

**Comment by rephus** [comment]
I'm focusing on other things right now, but feel free to submit a PR :)

**Comment by salvadorc17** [comment]
What do you mean, do in my own fork? Also code is using ajax for load roms? no other way to do that?
